### PR TITLE
docs: improve markdownlint and add rules

### PR DESCRIPTION
**Pull request by aepfli**
Changing from markdownlint-cli to markdownlint-cli2.

Changing configuration based on the cli

added custom rule, for max one sentence per line.

### CODE REVIEW

**Review by beeme1mr — APPROVED** [review]

**Review by justinabrahms — APPROVED** [review]

File: .markdownlint-cli2.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+config:
+    default: true
+    MD010:
+        spaces_per_tab: 4
+
+    MD013: false
```
[the remark this answers is not in the document]
Comment (aepfli) [reply .markdownlint-cli2.yaml:7]: i will replace all rule short codes with their naming alternative

[the remark this answers is not in the document]
Comment (aepfli) [reply .markdownlint-cli2.yaml]: i hope this is now better understandable :)

[the remark this answers is not in the document]
Comment (justinabrahms) [reply .markdownlint-cli2.yaml]: Am I missing it? The file didn't change?

[the remark this answers is not in the document]
Comment (aepfli) [reply .markdownlint-cli2.yaml]: Looks like a strange user experience, if you open the pull request and switch to the files tab, you normally should see it. It was she content of the last commit

**Pull request merged**